### PR TITLE
Measure runtime of different algorithm components

### PR DESCRIPTION
**Pull request by SebastianSchmidl**

### CODE REVIEW

File: distod/src/main/scala/com/github/codelionx/util/timing/AggregatingTiming.scala [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      val result = block
+      result
+    } finally {
+      addTiming(label, t0, System.nanoTime())
+    }
+  }
+
+  def print(): Unit = {
+    timings.toSeq
+      .sortBy(_._2)
```
Comment (CodeLionX) [inline distod/src/main/scala/com/github/codelionx/util/timing/AggregatingTiming.scala]: Codacy Issue found: [Found unnamed result or parameter](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=4846445)

Comment (CodeLionX) [inline distod/src/main/scala/com/github/codelionx/util/timing/AggregatingTiming.scala]: Codacy Issue found: [Found unnamed result or parameter](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=4846445)

Comment (CodeLionX) [inline distod/src/main/scala/com/github/codelionx/util/timing/AggregatingTiming.scala]: Codacy Issue found: [Found unnamed result or parameter](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=4846445)

Comment (SebastianSchmidl) [inline distod/src/main/scala/com/github/codelionx/util/timing/AggregatingTiming.scala:45]: Codacy Issue found: [Found unnamed result or parameter](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=4846445)

**Pull request merged**